op-guide/backup-restore.md

```diff
@@ -0,0 +1,116 @@
+---
+title: backup and restore
+category: advanced
+---
+
+# Backup and Restore
+
+## About
+
+This document describes how to back up and restore the data of TiDB. Currently, this document only covers full backup and restoration.
+
+Here we assume that the TiDB service information is as follows:
+
+|Name|Address|Port|User|Password|
+|:----:|:-------:|:----:|:----:|:------:|
+|TiDB|127.0.0.1|4000|root|*|
+
+In the process of backing up and restoring data, use the following tools:
```

Comment:
In the process of backing up and restoring data, use the following tools:
->
Use the following tools for data backup and restoration: